mmseg/models/backbones/twins.py

```diff
@@ -155,7 +155,8 @@ class LocallygroupedSelfAttention(BaseModule):
         attn_drop_rate (float, optional): Dropout ratio of attention weight.
             Default: 0.0
         proj_drop_rate (float, optional): Dropout ratio of output. Default: 0.
-        window_size(int): Window size of LSA. Default: 1.
+        window_size(int): Window size of LSA. LSA can be equal to GSA when
+            window_size=1. Default: 1.
```

Comment:
> LSA can be equal to GSA when window_size=1

NOT equal, when window_size=1, LSA without self-attention, while when reduction ratio is 1, GSA is full self-attention.